### DISCUSSION TITLE
Where are files stored?

### DISCUSSION THREAD

**Discussion by estebanmax** [TransformerOptimus/SuperAGI#321] [Q&A]
Thank you! But I want to point out two essential questions I didn't find in doc and comments:
1) after installing, I see my my disk space was decreased of about 7Gb!!! But where are all these files been installed? and so how to unistall SuperAGI??
2) how to activate it again  (I mean the localhost connection) after closing it? Do I have to execute everytime "docker-compose"??

**Comment by FrancisVarga** [comment]
its writing in workspace but each agent should create an folder but somehow it doesnt. even telling the agent creating a separated folder.

**Reply by neelayan7** [reply]
You can add RESOURCES_OUTPUT_ROOT_DIR: workspace/output/{agent_id} in the config.yaml

**Comment by sainsay** [comment]
The current docker-compose.yaml has a volume for the postgress database. This volume gets created somewhere on you machine depending on your docker configuration. For instance, the default Docker Desktop with wsl enable places it here: `C:\Users\{user}\AppData\Local\Docker\wsl\data`

This volume can be quite big again depending on your configuration.

For Docker Desktop the location can be changed in Settings->Resources->Disk image location
